### ISSUE TITLE
Using LM with Jobs Reborn

### ISSUE THREAD

**Issue by ThomasLETELLIER** [lokka30/LevelledMobs#362]
**Is your feature request related to a problem? Please describe.**
The plugin is great, but unfortunately it penalizes people who use Jobs Reborn, because the monsters are more powerful (and therefore the money/exp gain remains the same, even with a monster that has 100HP more)

**Describe the solution you'd like**
- The possibility to check the level of the player with %jobsr_user_jlevel_(hunter)% (%jobsr_user_jlevel_(jobname/id)%): This way, the higher the person is in this level, the more powerful the monsters spawn.
- Ability to run a specific command for EXP / Money gains via Reborn, linked to the player's level

**Describe alternatives you've considered**
I almost found a workaround, unfortunately I can't check if the player in question is a Hunter or not

**Additional context**
It should be taken into consideration that a player might not be a Hunter for the rewards

**Comment by lokka30** [comment]
I think I can add indirect integration for Jobs in LevelledMobs v4.0.0 by having a custom boolean evaluator that utilizes PlaceholderAPI placeholders. This means you can make a custom condition that requires the player to have the hunter job.
```yaml
player-levelling:
   ...strategy config here...
   condition: '%player-name%=lokka30`
# Only players that have the name 'lokka30' will
# be able to utilize player levelling.
```

**Comment by lokka30** [comment]
Closed as #375 will make this possible in LevelledMobs v4.0.0. @ThomasLETELLIER
